scripts/docs/templates/pkg.tpl

```diff
@@ -0,0 +1,58 @@
+{{ define "packages" }}
+---
+title: "API reference"
+description: "Prometheus operator generated API reference docs"
```

Comment:
Prometheus?

Reply:
> Prometheus?

Copied from the Prometheus operator 🤣 I'll fix it.